.editorconfig

```diff
@@ -163,7 +163,7 @@ dotnet_code_quality.ca2208.api_surface = public
 dotnet_diagnostic.RS0037.severity = none
 
 # License header
-file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.\n
+file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.
```

Comment:
LICENSE file seems correct according to guidelines. 
I'm not sure about the header, but all other dotnet repos are using same (runtime, aspnetcore). I'll ask around but I think it would be ok to change it in another PR if needed.

Reply:
It is inconsistent with SDK: https://github.com/dotnet/sdk/blob/1d4cb25ff11a718bece0bdd540d6ba1a6f86a719/src/Cli/dotnet/commands/dotnet-pack/PackCommand.cs#L1-L2

Reply:
The guidance about copyrights is here: https://github.com/dotnet/runtime/blob/main/docs/project/copyright.md 
According to it, the current header that was applied in this PR is correct.